doc/py_tutorials/py_feature2d/py_fast/py_fast.markdown

```diff
@@ -108,18 +108,18 @@ kp = fast.detect(img,None)
 img2 = cv2.drawKeypoints(img, kp, None, color=(255,0,0))
 
 # Print all default params
-print "Threshold: ", fast.getThreshold()
-print "nonmaxSuppression: ", fast.getNonmaxSuppression()
-print "neighborhood: ", fast.getType()
-print "Total Keypoints with nonmaxSuppression: ", len(kp)
+print( "Threshold: ", fast.getThreshold() )
```

Comment:
Without `from __future__ import print_function` in python 2 the result will be like `('Threshold: ', '12345')`, so it's better to add this import to avoid such output mismatches between python 2/3.